src/general/OptionsMenu.cs

```diff
@@ -1058,7 +1060,15 @@ private void OnCustomUsernameTextChanged(string text)
 
     private void OnLanguageSettingSelected(int item)
     {
-        Settings.Instance.SelectedLanguage.Value = languageSelection.GetItemText(item);
+        languages = TranslationServer.GetLoadedLocales().Cast<string>().OrderBy(i => i, StringComparer.InvariantCulture)
+            .ToList();
+        List<string> languageList = new List<string>();
+        foreach (var locale in languages)
+        {
+            languageList.Add(locale);
+        }
+
+        Settings.Instance.SelectedLanguage.Value = languageList[item];
```

Comment:
wohah, why did you do that so complicated?
languages already holds the loaded locales sorted, therefore you don't need line 1063-1064
in lines 1065-1069 you copy over the contents of `languages` into `languageList` and used `languageList`. Why didn't you just use the `languages` list directly?
```suggestion
        Settings.Instance.SelectedLanguage.Value = languages[item];
```

Reply:
I thought i needed to reset languages but i was wrong, sorry for making it more complicated than it should have been.
I made all the changes you suggested, thanks for your review